lib/ecto_cursor_based_stream.ex

```diff
@@ -102,22 +105,13 @@ defmodule EctoCursorBasedStream do
   end
 
   defp get_rows(repo, query, cursor_field, cursor, max_rows) do
-    import Ecto.Query
-
     query
+    |> _where(cursor_field, cursor)
```

Comment:
minor: I preferred it as it was, because now `_where` fn name doesn't really describe what it does

- `_` prefix is unneeded in Elixir for private functions, as it's already known they are private thanks to `defp`
- `filter_by_cursor` or `where_after_cursor` `where_cursor_larger_than` would be more readable function names, describing what it does